<launch>
  <arg name="model_name" default="albatross"/>

  <include file="$(find tauv_vehicle)/launch/thrusters/thrusters.launch">
    <arg name="model_name" value="$(arg model_name)" />
  </include>

  <node pkg="tauv_vehicle"
        type="oakd"
        name="oakd_front"
        output="screen"
  >
    <param name="frame" value="oakd_front" />
    <param name="ip" value="" />
  </node>
</launch>
